inc/html.class.php

```diff
@@ -1355,6 +1355,10 @@ static function includeHeader($title = '', $sector = 'none', $item = 'none', $op
          if (in_array('cable', $jslibs)) {
             Html::requireJs('cable');
          }
+
+         if (in_array('socket', $jslibs)) {
+            Html::requireJs('socket');
+         }
```

Comment:
There is no need to have 2 different names for this. If it is removed, you have to also remove the `case 'socket':`.
```suggestion
```

Reply:
agree with that